packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -2698,6 +2698,33 @@ void main() {
 
     expect(tester.getTopLeft(find.byWidgetPredicate((Widget w) => '${w.runtimeType}' == '_PopupMenu<int?>')), const Offset(8.0, 90.0));
   });
+
+  testWidgets("PopupMenuButton icon inherits IconTheme's size", (WidgetTester tester) async {
```

Comment:
It would also be good to test explicitly setting the icons size in the widget, and the 24 pixel fallback.

Reply:
Done)